### ISSUE TITLE
[Deleted]

### ISSUE THREAD

**Issue by LittlePooh-Git** [iblobtouch/iblobtouch.github.io#75]
[Deleted]

Edit 2: Deleted everything.

**Comment by lucasburlingham** [comment]
Instead of being slightly annoying, please ask for specific features or fixes. If you would like, I would expect that the developer would love to review a pull request.

**Comment by SCPlayz7000** [comment]
> Instead of being slightly annoying, please ask for specific features or fixes. If you would like, I would expect that the developer would love to review a pull request.

Agreed. Also, this isn't a game. This is a modding tool collection. If you want to play the game, go to https://arras.io/#wpd on sandbox mode

**Comment by xXsansoloXx** [comment]
I'm so sorry for this. I was like 13 when I made this, so I was very immature. I honestly didn't even know I made a post like this ^^' I just saw this in my email and thought it was someone elses thread I commented on. Again, I'm so sorry for this.
